native/acter/src/api/room.rs

```diff
@@ -1002,7 +1002,7 @@ impl Room {
                 let event_content = evt.event.deserialize_as::<RoomMessageEvent>()?;
                 let original = event_content
                     .as_original()
-                    .expect("Couldn't get original msg");
+                    .expect("Original msg should be got from event content");
```

Comment:
this is not more clear, leave the earlier. But also move it from `expect` to `context` probably...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1252/commits/8d49b591e076f69a6611b094d2dbe73e0ca1db8e